gui/elements/qt/_imageLayer.py

```diff
@@ -0,0 +1,52 @@
+from PyQt5.QtWidgets import QSlider, QComboBox, QHBoxLayout, QGroupBox, QVBoxLayout, QCheckBox
+from PyQt5.QtCore import Qt
+
+class QtImageLayer(QGroupBox):
+    def __init__(self, layer):
+        super().__init__(layer.name)
+        self.layer = layer
```

Comment:
Not super important but we should be using either `weakref.ref` with a `property` or `weakref.proxy` to avoid circular references keeping the objects from being garbage collected.

Reply:
Ok, I don't really understand this one (will have to look it up). Maybe we'll revisit after I try making a base class.

Reply:
done